ChangeLog

```diff
@@ -37,6 +37,10 @@ Release date: TBA
 
   Closes #5281
 
+* Fixed false negative ``unpacking-non-sequence`` when value is an empty list.
```

Comment:
The changelog entry in `2.13` is missing

Reply:
Forgive me if I'm missing something obvious but this is the 2.13 Changelog, right?

Reply:
We're using two separate documents to track changelog entries (don't ask me why.. 😅).
There is `Changelog` and `doc/whatsnew/2.13`, I think the latter is used to update the changelogs on the documentation pages.

Reply:
> We're using two separate documents to track changelog entries (don't ask me why.. 😅).

Well that's a good question though 😄 Unifying the two would take some time but the benefit of having only one place to modify would be great. Black has pretty great check on changelog entry, we may take example. Do you want to open the issue as you had the idea @DanielNoord or should I do it ?

Reply:
Feel free to open the issue @Pierre-Sassoulas

Reply:
Opened #5728